packages/greencheck/tests/SitecheckValidationTest.php

```diff
@@ -16,6 +16,9 @@ class SitecheckValidationTest extends TestCase
      */
     protected $sitecheck = null;
 
+    protected $redis = null;
```

Comment:
you only need to add this if you assign something to it. So using $this->redis = ... 
As you don't use it, you only use a local variable in the setUp method, this variable is not needed.